rclcpp/test/rclcpp/node_interfaces/detail/test_template_utils.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
I realize this might be based on another test file (copy-paste-edit), but personally, I think the copyright year should be 2022, since all the substantial content is new (and written this year).